Tell autoconf about with-plugin option to avoid warnings like:
configure: WARNING: unrecognized options: --with-plugin-maria

Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

Index: mariadb-5.1.67/config/ac-macros/plugins.m4
===================================================================
--- mariadb-5.1.67.orig/config/ac-macros/plugins.m4
+++ mariadb-5.1.67/config/ac-macros/plugins.m4
@@ -38,6 +38,7 @@ AC_DEFUN([_MYSQL_PLUGIN],[
   _MYSQL_PLUGAPPEND([__mysql_plugin_list__],[$1])
   m4_define([MYSQL_PLUGIN_NAME_]AS_TR_CPP([$1]), [$3])
   m4_define([MYSQL_PLUGIN_DESC_]AS_TR_CPP([$1]), [$4])
+  m4_ifdef([_AC_ENABLE_IF], [_AC_ENABLE_IF([with],[plugin-$1])])
   _MYSQL_PLUGAPPEND_META([$1], $5)
   ifelse(m4_bregexp(__mysql_include__,[/plug\.in$]),-1,[],[
      MYSQL_PLUGIN_DIRECTORY([$1],
